src/Sarif.Driver/Sdk/Errors.cs

```diff
@@ -238,31 +238,34 @@ public static void LogExceptionLoadingPlugIn(string plugInFilePath, IAnalysisCon
         public static void LogUnhandledRuleExceptionAssessingTargetApplicability(
             HashSet<string> disabledSkimmers,
             IAnalysisContext context,
-            Exception ex)
+            Exception exception)
         {
             string ruleId = context.Rule.Id;
             string ruleName = context.Rule.Name;
             context.Rule = Errors.RuleDisabled;
 
+            Result result = RuleUtilities.BuildResult(ResultKind.InternalError, context, null,
+                                nameof(SdkResources.ERR0998_ExceptionInCanAnalyze),
+                                ruleName,
+                                exception.ToString());
```

Comment:
Yes, I think it makes sense not to recapitulate this information if we're producing a stack. Will fix.